content/docs/esc/reference/well-known-properties/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Well-Known Properties
+title_tag: Well-Known Properties
+h1: Well-Known Properties
+meta_desc: Pulumi ESC allows you to compose and manage hierarchical collections of configuration and secrets and consume them in various ways.
+menu:
+  esc:
+    parent: esc-reference
+    identifier: esc-ref-well-known-properties
+    weight: 7
+---
+
+The [`esc` CLI](/docs/install/esc/) and other ESC consumers (e.g. the [`pulumi` CLI](/docs/install/)) conventially assign specific semantics to certain top-level properties of evaluated ESC environments.
+
+## Properties
+
+- [`environmentVariables`](/docs/esc/reference/well-known-properties/environment-variables)
```

Comment:
What about pulumiConfig?

Reply:
`pulumiConfig` is not interpreted by the `esc` CLI, so I opted to leave it out. It's also already documented in the Pulumi IaC integration reference. I don't feel super strongly about this one, though, so if folks think it should be in here I'll add it.

Reply:
I think it would be expected here, they may not have seen the Pulumi IaC reference. You also mentioned "other consumers" like IaC at the top of the file.

Reply:
> I think it would be expected here, they may not have seen the Pulumi IaC reference. 

SG. I'll add some headers to differentiate "always expected" vs. "only expected when integrating with tool x".

> You also mentioned "other consumers" like IaC at the top of the file.

Yeah--what I was trying to communicate is that _any_ consumer of an evaluated environment that deals with envvars, etc. should probably interpret these properties.

Reply:
> `pulumiConfig` is not interpreted by the `esc` CLI, so I opted to leave it out. It's also already documented in the Pulumi IaC integration reference. I don't feel super strongly about this one, though, so if folks think it should be in here I'll add it.

I would mention it and say exactly that (in so many words) in the docs!